### ISSUE TITLE
Master Looter bug

### ISSUE THREAD

**Issue by ciccioriccio** [dalaranwow/dalaran-wow#3462]
Some items like Mounts and Items that starts a quest may bug and u cant loot them.
If u are in a raid and the loot sistem is set on Master Looter, items like the Mount in VoA or the Head of Onyxia in the respective raids cant bee looted by the master looter and neither buy the others raid members, not even if u change loot system to free for all, need before greed or group loot.
I's working fine with a different loot sistem like Group loot, but it may be dangerous for the so called ninja.

**Comment by ashido96** [comment]
Will confirm, master loot will bug the head of onyxia and no one will be able to loot it.

**Comment by Rushor** [comment]
this can happen to any kind of item no?

**Comment by ImprezaWoW** [comment]
Yes, happens a lot to any item during raids, usually when looting the last boss after a player quits raid. So, happens with KT a lot in Naxx too.

**Comment by ciccioriccio** [comment]
its not correct imprezaWoW, there are two different bug. the one that i've posted here and an other one. this one u are talking about is a bug over all kind of loot, but it is still about Master looter. In that case if the man who open the loot leaves the raid, no one in the raid can be able to see or loot the items inside the mob/boss. only if u invite him back it can be "fixed", but several time it is pretty hard or completely impossible to know who was the first person to open the loot.

**Comment by lildevilre** [comment]
ImprezaWow, Im sorry but it seems that proof is not good enough....... 
![wowscrnshot_062516_202158](https://cloud.githubusercontent.com/assets/20197610/16438237/d6195ccc-3d7b-11e6-8ee3-49c3487cc735.jpg)

**Comment by lildevilre** [comment]
or 
![wowscrnshot_062516_202201](https://cloud.githubusercontent.com/assets/20197610/16438367/1d379208-3d7d-11e6-8ac8-37784b80eb12.jpg)

**Comment by Golganeth** [comment]
This report has nothing to do with "ninjaing", in fact, bug tracker isn't used to report players or similar. Please read the report and stay on-topic, and add relevant info to the bug report if you have any.

What Norganon said is the truth, Game Masters will not assist with the so-called "ninjaing" situations, community should moderate itself, where ninjas are shunned by Dalaran citizens. Be careful with the raids you join and consider running with Master Loot if valuable items are likely to drop.

If you, however, have any proof or evidence where the raid leader clearly states the loot rules beforehand, we can then take action for "scamming". 

Regards

**Comment by lildevilre** [comment]
i didnt use it to report a player, and i provided the correct info with screenshots. What I was in reference to was and I quote "you have evidence of the rules beforehand and if anyone ninjas an item, you will have proof of the wrongdoing, AND the loot won't bug out."  the phrase in the sentence was you will have proof, as though something would be done. When by my understanding you all do nothing about it. so whats the point in having proof? The statement shouldnt have been made about "proof of the wrongdoing" if you all do nothing with it.

**Comment by lildevilre** [comment]
Well, lets start here, #1, I did not use bugtracker to report a player #2, I did go through the proper channels and did have the correct screenshots. #3, I didnt fall off topic. I simply stated that the information given was incorrect, and then provided proof of why it was incorrect. #4, I stated that the proof was not good enough, and that may not seem as though it adds no relevant information to you it might save the next GM from having to answer the ticket about a "Ninja" and allow you all to provide better service to people in need of help for questing or raiding with actual problems. I did not post this to cause controversy, I simply posted so that the community as a whole would be more clear on the rules in regards to the topic of Master Looter vs Master Ninja and be able to provide the correct and proper proof for something to be done instead of being stuck in my situation where nothing can be done.

**Comment by Golganeth** [comment]
> I didnt fall off topic.

You did. This is a bug report about Master Looter / loot getting stuck.
@ImprezaWoW just mentioned some workarounds to help relieve this problem (change master looter to group loot, explain rules beforehand so that we, GMs, can take action if "scamming" happens - someone needing on a random item...)

> I simply stated that the information given was incorrect, and then provided proof of why it was incorrect.

The information given wasn't incorrect:
>@ImprezaWoW: In that case, you have evidence of the rules beforehand and if anyone ninjas an item, you will have proof of the wrongdoing,

As said above, if a raid leader especifically explains the loot rules (e.g MS>OS - or highest roll wins) and then after that he/she takes it anyway, then we can take action for "scamming". So always ask for the Loot Rules beforehand.

-----------

If you actually wanted to save us some work, you could just mention this on the [Community Support](http://www.dalaran-wow.com/forums/input-and-assistance/community-support) or [General Discussion](http://www.dalaran-wow.com/forums/community/general-discussion) sections of the forums. This is not the correct place, this is the bug tracker and it's a bug report about Master Looter and loot issues. 

You are not saving us work but giving us some extra by having to explain this to you. (I might clean this mess from github later).

Thanks!

**Comment by Golganeth** [comment]
> I didnt fall off topic.

You did. This is a bug report about Master Looter / loot getting stuck.
@ImprezaWoW just mentioned some workarounds to help relieve this problem (change master looter to group loot, explain rules beforehand so that we, GMs, can take action if "scamming" happens - someone needing on a random item...)

-----------------------

> I simply stated that the information given was incorrect, and then provided proof of why it was incorrect.

The information given wasn't incorrect:
>@ImprezaWoW: In that case, you have evidence of the rules beforehand and if anyone ninjas an item, you will have proof of the wrongdoing,

As said above, if a raid leader especifically explains the loot rules (e.g MS>OS - or highest roll wins) and then after that he/she takes it anyway, then we can take action for "scamming". So always ask for the Loot Rules beforehand.

-----------
---------------

If you actually wanted to save us some work, you could just mention this on the [Community Support](http://www.dalaran-wow.com/forums/input-and-assistance/community-support) or [General Discussion](http://www.dalaran-wow.com/forums/community/general-discussion) sections of the forums. This is not the correct place, this is the bug tracker and it's a bug report about Master Looter and loot issues. 

You are not saving us work but giving us some extra by having to explain this to you. (I might clean this mess from github later).

Thanks!

**Comment by Golganeth** [comment]
The mess was cleaned. @lildevilre thanks for deleting your comments, I hope you understood what I was saying. We will take action for "scamming" if there's enough evidence of a raid leader explaining the loot rules, and after that, he/she takes an item anyway. So always ask for loot rules beforehand.

**Comment by lildevilre** [comment]
Golganeth, Yes I have seen first hand the actual difference. Thanks for explaining a bit better and making things more clear!

**Comment by Rushor** [comment]
#[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**:

Hi. I hope I'm not making a duplicate post, but I see there's a problem with looting in almost all encounters (raids). I experienced one myself today on a Obsidium Sanctum 3D run. After the Raid Leader used Master Loot to distribute most of the drop, he left before he traded the final pieces of gear. Because of this, I got screwed on a piece of gear (mainly the gloves : Gloves of the Lost Vanquisher; 
http://www.wowhead.com/item=40615/gloves-of-the-lost-vanquisher). Morengs (GM) tried to help me ingame, but I was still forced to send a mail and now I'm waiting for a reply and hopefully the item (MrAnderson2008@yahoo.com is my email).
I got lucky this time (was the only one in the 10 man raid under 5k gs) and I doubt I'll have the time/luck again as I did today.

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

If the raid leader leaves suddenly, or if alot of raiders leave, the loot sometimes bug. I have no way to say exactly how it happens or in what conditions or if it happens for all the raids or not. It definitely happens for Sartharion tho.

**Expected behaviour**: Tell us what should happen instead

The loot should be lootable even if some of the raiders leave or even the raid master looter or raid leader.

**Steps to reproduce the problem**:

1.  Join a raid.
2.  Tell raiders to leave after the boss is down, or tell the raid/loot master to leave.
3.  Try to loot.

**Include proofs for this behaviour**

https://postimg.org/gallery/2fgwq2z2w/

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MrAndersonX** [comment]
Yeah it definitely needs a fix !

**Comment by Chenocide** [comment]
This is still a problem and not just with Master loot. It can happen as long as someone is outside of the raid when something is killed/loot screen being opened. An example is the cores that you loot on Mimiron in order to bring down the head in P3. You can have someone in the raid group but not inside of the raid itself, and the loot can bug with free for all loot on.
